src/window.cc

```diff
@@ -29,8 +30,8 @@ void Window::generate_keybindings() {
   }
 }
 
-Window::Window() : box(Gtk::ORIENTATION_VERTICAL), notebook(directories), compiling(false) {
-  DEBUG("start");
+Window::Window() : box(Gtk::ORIENTATION_VERTICAL), notebook(*Singleton::directories()), compiling(false) {
```

Comment:
No need to pass directories to notebook anymore, since directories are now a Singleton